packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -800,55 +774,28 @@ describe("BlobManager", () => {
 			assert.strictEqual(summaryData.redirectTable, undefined);
 		});
 
-		// tests results will change after
```

Comment:
This test is now not needed since we don't resolve on disconnection. It becomes the same as the one checking abort on reconnect